rules-tests/DeadCode/Rector/Property/RemoveUselessVarTagRector/Fixture/remove_null.php.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\Property\RemoveUselessVarTagRector\Fixture;
+
+final class RemoveNull
+{
+    /**
+     * @var null
+     */
+    public $name = null;
+}
+
+?>
+----
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\Property\RemoveUselessVarTagRector\Fixture;
+
+final class RemoveNull
+{
+    public $name = null;
```

Comment:
This also make phpstan notice no type specified.

I think instead of remove `null` docblock, there should be a rule to resolve the type, then it can be properly removed.

Reply:
Agreed. That's responsibility of another rules.